exercises/concept/census/.docs/instructions.md

```diff
@@ -0,0 +1,96 @@
+# Instructions
+
+It is your job to prepare the city's IT system for an upcoming census. Specifically, you are responsible for the program that will process the data from the census workers.
+
+The program must be able to create a new resident in the system when given a resident's information. Additionally, you will create functions that ensure the required information is present in the resident's data and delete a resident's data. Lastly, you will count the residents to provide an accurate census count.
+
+## 1. Create a new resident
+
+When a census worker collects a resident's information, they need to register that resident by entering their name, age, and address into the system.
+
+Implement the function `NewResident` that accepts three arguments:
+
+- The name of the resident.
+- The age of the resident.
+- The address of the resident.
+
+The function should return a pointer to a `Resident` struct that holds this information.
+
+```go
+name := "Matthew Sanabria"
+age := 29
+address := map[string]string{"street": "Main St.",}
+
+NewResident(name, age, address)
+// => &{Matthew Sanabria 29 map[street:Main St.]}
+```
+
+## 2. Validate required information
+
+Residents may be reluctant to provide personal data to census workers. In those cases it's necessary to determine if the resident provided the required information to be counted in the census.
+
+In order to be counted, a resident must provide a non-zero value for their name and an address that contains a non-zero value for the `street` key. All other information, such as the resident's age, is optional. Implement the `HasRequiredInfo` method that returns a boolean indicating if the resident has provided the required information.
+
+```go
+name := "Matthew Sanabria"
+age := 0
+address := make(map[string]string, 0)
```

Comment:
`make` should be only called with 1 argument in case of map

Reply:
In this case, yep!